@navikt/core/react/src/button/button.stories.tsx

```diff
@@ -278,3 +279,37 @@ export const LoadingWithAs = {
   args: { chromatic: { disableSnapshot: true } },
   chromatic: { disableSnapshot: true },
 };
+
+export const LoadingDemo = () => {
+  const [l, setL] = useState(false);
+
+  const handleLoading = () => {
+    setL(true);
+    setTimeout(() => setL(false), 2000);
+  };
+
+  return (
+    <Tabs defaultValue="logg">
+      <Tabs.List>
+        <Tabs.Tab value="logg" label="Logg" />
+        <Tabs.Tab value="inbox" label="Inbox" />
+      </Tabs.List>
+      <Tabs.Panel value="logg" className="h-24 w-full bg-gray-50 p-4">
+        <VStack gap="6">
+          <button onClick={handleLoading}>Toggle loading</button>
+          <Button loading={l} onClick={handleLoading}>
+            Testknapp 1
+          </Button>
+        </VStack>
+      </Tabs.Panel>
+      <Tabs.Panel value="inbox" className="h-24 w-full bg-gray-50 p-4">
```

Comment:
Hva er formålet med to like eksempler?

Reply:
Når man byttet mellom tabs, ble button sin tekst automatisk lest opp av voiceover. Dette var en av feilene da button-tekst alltid ble lest opp når knapp gikk fra å være i en container med `display: none` til synlig